tools/cli/admin.go

```diff
@@ -665,6 +665,20 @@ func newAdminTaskListCommands() []cli.Command {
 				AdminDescribeTaskList(c)
 			},
 		},
+		{
+			Name:    "list",
+			Aliases: []string{"ls"},
```

Comment:
nit: probably just "l" to match the alias of other commands like `wf l`, of `domain l` ?